daemon/buildinfo.c

```diff
@@ -200,8 +200,67 @@
 
 #define FEAT_YES_NO(x) ((x) ? "YES" : "NO")
 
+// coverity[ +tainted_string_sanitize_content : arg-0 ]
+static inline void coverity_remove_taint(char *s)
+{
+    (void)s;
+}
+
+void read_install_type(void) {
+    char *filepath;
+    filepath = mallocz(sizeof(char) * (strlen(netdata_configured_user_config_dir) + strlen(".install-type") + 2));
+    sprintf(filepath, "%s/%s", netdata_configured_user_config_dir, ".install-type");
```

Comment:
please, prefer to use `snprinfz` like this [code](https://github.com/netdata/netdata/blob/6600f33247368f0b124351abe5e9acf6e49c8cd2/daemon/analytics.c#L387), this is minor.

Reply:
not sure this is necessary as `netdata_configured_user_config_dir` is not likely to change contents between lines 211 & 212

Reply:
@underhood is correct, both things being used here are constant at runtime (barring memory corruption, but that’s beyond unlikely in this case).